### ISSUE TITLE
Can't add environment variables to schema.mjs

### ISSUE THREAD

**Issue by eddyhdzg-solarx** [t3-oss/create-t3-turbo#95]
When adding to serverSchema an environmental variable, it fails.

```ts
export const serverSchema = z.object({
  NODE_ENV: z.enum(["development", "test", "production"]),
  DATABASE_URL: z.string().url(), // New env variable
});
```

I get the following error:

```
@acme/nextjs:dev:  DATABASE_URL: Required
@acme/nextjs:dev: 
@acme/nextjs:dev: error - Failed to load next.config.mjs, see more info here https://nextjs.org/docs/messages/next-config-error
@acme/nextjs:dev: Error: Invalid environment variables
```

**Comment by thlmenezes** [comment]
This is the expected behavior, it blocks the next initialization when the environment variables do not meet the schema.

* If you're receiving this error **locally**:

check your `.env` file directory, next will search for such file inside its directory. My sugestion would be to use a symbolic link, check code below

```bash
ln -s `pwd`/.env apps/nextjs/.env
```

* If you're receiving this error on **staging/production**:

check your environment variables, `DATABASE_URL` does not exist there

**Comment by juliusmarminge** [comment]
Fixed in #100 by running nextjs with the same `with-env` wrapper as the `db` package.
